documentation/introduction/introduction-concepts.asciidoc

```diff
@@ -0,0 +1,83 @@
+---
+title: Core Concepts
+order: 2
+layout: page
+---
+= Core Concepts
+
+In the following, we go through some concepts central to Vaadin development.
+
+== Rapid Web Application Development
+
+At Vaadin, we pride ourselves on developer productivity.
+We feel that developers should be able to forget the framework they run on, so that they can concentrate on developing the best application they can.
+
+For rapid application development, Vaadin offers:
+
+* Customizable project templates in link:https://vaadin.com/start/latest[Get Started with Vaadin]
+* Downloadable application examples
+* Maven archetypes
+* Eclipse plugin for rapid project creation
+* Vaadin Designer for interactive UI designing
```

Comment:
Since each of these are on a separate page on our site, I think it makes sense to collect them here, maybe even with links? Users that jump immediately to the getting started documentation will otherwise miss these completely. The text can be re-worded to have less of that marketing feel :)

Reply:
Yeah, that indeed a good idea, that our documentation would have some sort of index of all the things we have on offer. Not sure if we have documentation about how to use our Maven archetypes, but for the others we have documentation we could link to.

I’m seeing something like this in the future, similar kind of starting page for our docs: https://ionicframework.com/docs